spree/core/app/models/concerns/spree/has_custom_fields.rb

```
@@ -181,11 +233,11 @@ def apply_custom_field_attrs(attributes)
         attributes.each_with_index do |raw, index|
           attrs = raw.respond_to?(:to_h) ? raw.to_h : raw
           attrs = attrs.with_indifferent_access
-          definition_id = attrs[:metafield_definition_id] || attrs[:custom_field_definition_id]
+          definition_id = attrs[:custom_field_definition_id] || attrs[:custom_field_definition_id]
```

RuboCop (1.88.2)

[warning] Binary operator `||` has identical operands.

(Lint/BinaryOperatorWithIdenticalOperands)